docs/docs/Configuration/configuration-authentication.md

```diff
@@ -103,12 +104,70 @@ LANGFLOW_SUPERUSER_PASSWORD=securepassword
 
 ### LANGFLOW_SECRET_KEY
 
-This environment variable holds a secret key used for encrypting the superuser's password. Make sure to set this to a secure, randomly generated string.
+This environment variable holds a secret key used for encrypting the superuser's password.
 
 ```bash
-LANGFLOW_SECRET_KEY=randomly_generated_secure_key
+LANGFLOW_SECRET_KEY=dBuuuB_FHLvU8T9eUNlxQF9ppqRxwWpXXQ42kM2_fb
+```
+
+Langflow uses the [Fernet](https://pypi.org/project/cryptography/) library for secret key encryption.
+
+### Create a LANGFLOW_SECRET_KEY
+
+The `LANGFLOW_SECRET_KEY` is used for encrypting sensitive data like API keys. It must be:
+- At least 32 bytes long
+- URL-safe base64 encoded
+
+If no key is provided, Langflow will automatically generate a secure key. This is not recommended for production environments.
```

Comment:
I think this could do with some expansion in explanation. My understanding is that the global variables (including secrets) are stored in the common Postgres DB, and so if you have multiple Langflow instances running (say in a Kubernetes context), they would have to have `LANGFLOW_SECRET_KEY` set to the same value? 

Consequently the generated key should be treated as one might an application access token: keep in a safe place such as a keysafe (and not committed code), etc.